README.md

```diff
@@ -1,2 +1,46 @@
-# satel-registry-push
-GitHub actions to push a docker image to registry 
+# Satel Docker Build Push
+
+This centralized GitHub action builds and pushes a docker image to registry 
+
+## Usage 
+```yml
+name: "Deploy webapp"
+on:
+  pull_request:
+    types:
+      - opened
+  push:
+    tags:
+      - "*"
+    branches:
+      - main  
+    
+    build-server:
+    name: Build server
+    needs: [generate-variables, build-client]
+    runs-on: <host_name>
+    # Map a step output to a job output, used in other jobs
+    outputs:
+      clean-branch-name: ${{ steps.registry-build-push.outputs.clean-branch-name }}
+    steps:    
+      - name: Build & Push server image to registry
+        id: registry-build-push
+        uses: SatelCreative/satel-docker-build-push@v1
+        with:
+          app-name: <app-name>
+          satel-docker-user: ${{ secrets.SATEL_DOCKER_USER }}
+          satel-docker-pass: ${{ secrets.SATEL_DOCKER_PASS }}
+          client-docker-user: ${{ secrets.CLIENT_DOCKER_USER }}
+          client-docker-pass: ${{ secrets.CLIENT_DOCKER_PASS }}
+          satel-registry: docker.satel.ca
+          client-registry: sb-docker.satel.ca
```

Comment:
Ok here is a place to simplify and also create more modularity in a way:
Don't define "satel" and "client" registries. Instead just have one docker registry defined and have the user handle the switch between the registries.
Typically we'll push the latest commit on main to the Satel registry for QA and a release/tag to the client's registry for staging and production, so the user can or even will have to setup 2 workflows. They can switch the docker registry.
This way this action is simpler, doesn't enforce/assume anything, and then the workflow is more modular/flexible.
Is there maybe even an action that already does what we need here if it's super general?

Reply:
hmm, I pass them both but only push to [client's if it's a tag ](https://github.com/SatelCreative/satel-docker-build-push/blob/9132355a6789a93cec37ef2bb8f495e0c542351c/DeployDockerImage.sh#L41). 
I am not sure how we will get the user to do that switch. I'll into it.

Reply:
You did two different workflow for tag and main for JD's project ... Why not the same for that?

Reply:
True, I think this is what you had in mind when you said we can probably replace this action completely now 
https://github.com/marketplace/actions/docker-build-push-action

Reply:
I guess we'd rely on the QA environment to see if the docker image is working properly.
It's fine if we get notified of failures right away on the main branch jobs.

Reply:
Ah brilliant. Then we should really switch to a generic public one instead of reinventing the while!
The one thing is that we need the Dockerfile to be production only in that case ...
We really to reorganize it all.